utils/convert.js

```diff
@@ -64,7 +67,7 @@ function makeLibFiles(sourceMaps) {
         plugins: [],
         presets: [
             [ '@babel/preset-env',
-              { modules: 'commonjs' } ]
+              { modules: false } ]
         ],
```

Comment:
The presets is doing some weird stuff with the code. We likely want to drop this (if we don't end up dropping babel completely).